mvlearn/model_selection/validation.py

```diff
@@ -0,0 +1,56 @@
+"""Validation utils."""
+# Copyright 2019 NeuroData (http://neurodata.io)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Authors: Pierre Ablin
+import numpy as np
+
+from sklearn.model_selection import cross_validate as sk_cross_validate
+from sklearn.pipeline import Pipeline
+
+from ..utils import check_Xs
+from ..split import ConcatSplitter
+
+
+def cross_validate(estimator, Xs, y, *args, **kwargs):
+    r"""Evaluate metric(s) by cross-validation and also record fit/score times.
```

Comment:
check pep257

Reply:
This is a copy and paste from sklearn :(